src/error-display/helper.ts

```diff
@@ -0,0 +1,22 @@
+import { MediaWidths } from "../spec/media-spec";
+
+export interface ImagePathAttributes {
+    base: string;
+    md: string;
+    lg: string;
+    width: number;
+    height: number;
+}
+
+export const imgAttributeHelper = (
```

Comment:
for the purpose of code organisation, can you move this type and util to a namespace called `ErrorDisplayHelper` and export that namespace instead? partly to avoid potential naming conflicts between public utility functions

Reply:
updated and added jsdoc